docs/guide/localization/static.md

```diff
@@ -239,17 +239,91 @@ As a reminder, namespacing is primarily necessary for *installable modules* and
+### Core UI localization
 
-To localize Apostrophe's existing user interface strings, we need to create JSON files for the `apostrophe` namespace, which contains all of the phrases for the core UI of Apotrophe. To do that, add JSON files for the locales to a project-level `modules/@apostrophecms/i18n/i18n/apostrophe` directory. The name of the subdirectory determines the namespace. You can also do this in any other module as long as you use an `i18n/apostrophe` subdirectory within that module, keeping these localizations separate from your project-level strings.
+The Apostrophe user interface contains many registered strings, currently localized to English and a few other languages. We will be working to provide more localization files, but if you are interested in adding l10n files to your project for the UI, you are welcome to do that.
+
+To localize Apostrophe's existing user interface strings, we need to create JSON files for the `apostrophe` namespace, which contains all of the phrases for the core UI of Apostrophe. To do that, add JSON files for the locales to a project-level `modules/@apostrophecms/i18n/i18n/apostrophe` directory. The name of the subdirectory determines the namespace. You can also do this in any other module as long as you use an `i18n/apostrophe` subdirectory within that module, keeping these localizations separate from your project-level strings.
 
 Note that separately installed npm modules may use other namespaces, which you can see in their source code. You can create subdirectories for these as well, again keeping these localizations separate from your project-level strings.
 
 In each JSON file, copy the contents of [the Apostrophe core l10n file](https://github.com/apostrophecms/apostrophe/blob/main/modules/@apostrophecms/i18n/i18n/en.json) to get all the keys. You can then start translating each string.
 
 Of course, this can be a lot of work and would likely involve tracking down where strings are used for testing purposes. If you are interested in being part of translating the UI for a language that isn't supported yet, please contact us in [Discord](http://chat.apostrophecms.com) or at [help@apostrophecms.com](mailto:help@apostrophecms.com) so we can coordinate efforts and let the whole community benefit.
 
-### Localizing your own custom Apostrophe user interface modules
+### Localizing schema field labels
+
+An important part of the admin UI that often needs localization is the field labels in your schemas. These labels appear throughout the admin interface and should be presented in the editor's preferred language. You can localize these labels using namespaced keys in your schema fields:
```

Comment:
... Right, if you want these strings in the admin UI they must be namespaced so you can `browser: true` them later. Good call.